tex/main/MarkovChain/expressions_from_pi/main.tex

```diff
@@ -328,3 +326,180 @@ \subsubsection{Overall Waiting Time}
+where \(u > 0 \). Thus, the set of blocking states can be defined as:
+
+\begin{equation*}
+    S_b = \{(u,v) \in S \; | \; u > 0\}
+\end{equation*}
+ 
+In order to not consider patients that will be lost to the system, the set of accepting
+states needs to be taken into account. As defined in section \ref{recursive-waiting-time-ambulance},
+the set of accepting states is given by (\ref{eq:accepting_states_ambulance}):
+
+\begin{equation*}
+    S_A^{(a)}=
+    \begin{cases}
+        \{(u, v) \in S \; | \; u < M \} & \textbf{if } T \leq N\\
+        \{(u, v) \in S \; | \; v < N \} & \textbf{otherwise}
+    \end{cases}
+\end{equation*}
+
+For the waiting time formula the mean sojourn time for each state was considered,
+ignoring any arrivals. Here, the same approach is used but ignoring only ambulance
+arrivals. That is because for the waiting time formula, once a patient enters the
+hospital (i.e. starts waiting) any patient arriving after them will not affect their
+pathway. That is not the case for blocking time. When an ambulance patient is blocked,
+any \textit{other} patient that arrives will cause the blocked patient to remain 
+blocked for more time. Therefore, \textit{other} arrivals are considered here:
+
+\begin{equation}\label{eq:time_in_state_blocking_time}
+    c(u,v) = 
+    \begin{cases}
+        \frac{1}{\min(v,C) \mu}, & \text{if } v = C\\
+        \frac{1}{\min(v,C) \mu + \lambda^o}, & \text{otherwise}
+    \end{cases}
+\end{equation}
+ 
+In equation \ref{eq:time_in_state_blocking_time}, both service completions and 
+\textit{other} patient arrivals are considered. 
+Thus, from a blocked patient's perspective whenever the system moves from one state 
+to another it can either be because of a service being completed or from an arrival 
+of \textit{other} patient. 
+The following two expressions give the probabilities of these two scenarios:
+
+\begin{equation*}
+    p_s(u,v) = \frac{\min(v,C)\mu}{\lambda^o + \min(v,C)\mu}, \qquad
+    p_o(u,v) = \frac{\lambda^o}{\lambda^o + \min(v,C)\mu}
+\end{equation*}
+
+
+Having defined \( c(u,v) \) and \(S_b\) a formula for the blocking time that is
+expected to occur at each state can be given by:
+
+\begin{equation}\label{eq:blocking-time-at-each-state}
+    b(u,v) = 
+    \begin{cases} 
+        0, & \textbf{if } (u,v) \notin S_b \\
+        c(u,v) + b(u - 1, v), & \textbf{if } v = N = T\\
+        c(u,v) + b(u, v-1), & \textbf{if } v = N \neq T \\
+        c(u,v) + p_s(u,v) b(u-1, v) + p_o(u,v) b(u, v+1), & \textbf{if } u > 0 \textbf{ and } v = T \\
+        c(u,v) + p_s(u,v) b(u, v-1) + p_o(u,v) b(u, v+1), & \textbf{otherwise} \\
+    \end{cases}
+\end{equation}
+
+Unlike equations (\ref{eq:waiting-time-at-each-state-other}) and 
+(\ref{eq:waiting-time-at-each-state-ambulance}), equation 
+(\ref{eq:blocking-time-at-each-state}) cannot be solved recursively. 
+An algebraic approach will be used to solve this equation here. 
```

Comment:
Yeah algebraic isn't quite right. Arguably doing things recursively is algebraic.

I'm not entirely what the correct term is:

- Direct approach;
- Linear algebraic approach;

Reply:
I agree, algebraic may not be the right name for it. Should we settle for linear algebraic approach for now? (I think I like it slightly better). There may even be a third name that describes it a better though.

Reply:
Could even be `Solving the linear system approach`